src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,6 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public abstract boolean isMutating();
```

Comment:
Header comment is necessary for methods to be overriden.

Reply:
Requested changes have been made.

Reply:
The comments should be shifted to the method instead. Good that you have it as an abstract method, so the overriding is enforced. The comment content should be what this method does. Use the following format "return true if ..." for simplicity or you can use the standard java doc format.